### ISSUE TITLE
Image issue

### ISSUE THREAD

**Issue by haophan1996** [JideGuru/rich_editor#13]
Hello JideGuru,
First, thank you for your work, this package look amazing, but there is an issue on image, image can scroll, can you fix this please

**Comment by JideGuru** [comment]
Can you please add a little more detail so I can understand/reproduce this issue on my end? Thanks

**Comment by haophan1996** [comment]
> Can you please add a little more detail so I can understand/reproduce this issue on my end? [Thanks]

Yes, here is it [https://www.youtube.com/watch?v=9uYg1PxANp8](url) 
As you see the video, every time I tapped on image then scroll up and down, it does not work, can you please fix this issue, thank you very much

**Comment by JideGuru** [comment]
Thanks for the Video. I'll get back to you during the weekend

**Comment by haophan1996** [comment]
> Thanks for the Video. I'll get back to you during the weekend

Thank you so much

**Comment by haophan1996** [comment]
I have one questions is there something wrong with show/hide keyboard, hide keyboard works fine, cursor still shows, but once I hit show keyboard, cursor disappears, could you fix this, please
